### PR TITLE
feat(spaces): Add owner to the Space model.

### PR DESCRIPTION
**Pull request by VineetReynolds**
Owners are resolved from the UUIDs sent by the REST API, through
a lookup against a userID map maintained on the client.
The resolved owners can be then used in Angular templates in views.

### CODE REVIEW

**Comment by VineetReynolds** [comment]
This is borrowed mostly from planner in how users for WIs are resolved, but there is a possibility of doing it differently.

File: src/app/spaces/space.service.spec.ts [hunk 1]
```diff
@@ -44,7 +44,13 @@ describe('Service: SpaceService', () => {
...
           provide: WIT_API_URL,
           useValue: "http://example.com"
-        }
+        },
+        {
+          provide: AUTH_API_URL,
+          useValue: 'http://example.com/auth'
+        },
+        UserService,
+        Broadcaster
```
Comment (joshuawilson) [inline src/app/spaces/space.service.spec.ts]: Why did you import Broadcaster, I do see it being used. Additionally I'm not sure that you are using UserService either but at least it is used in another file.

Comment (VineetReynolds) [inline src/app/spaces/space.service.spec.ts]: The Jasmine tests failed compilation until I added those. I only needed UserService though as a transitive dep of SpaceService.

Comment (joshuawilson) [inline src/app/spaces/space.service.spec.ts]: ok, thanks

**Pull request merged**